TerrorConsole/Assets/_Main/Scripts/Camera/ICameraShake.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    public interface ICameraShake
+    {
+        public void CameraShake();
+    }
+}
```

Comment:
This interface should be removed completely, the camera system should drive the shaking.